diff --git a/src/MapPoint.php b/src/MapPoint.php
index bc3c753..e118b26 100644
--- a/src/MapPoint.php
+++ b/src/MapPoint.php
@@ -19,14 +19,14 @@ class MapPoint extends Field
     /**
      * Resolve the field's value.
      */
-    public function resolve($resource, ?string $attribute = null): void
+    public function resolve($resource, $attribute = null): void
     {
         parent::resolve($resource, $attribute = null);
         $this->latlng = $this->geometryToLatLon($this->value);
         $this->withMeta(['latlng' => $this->latlng]);
     }
 
-    public function fillModelWithData(object $model, mixed $value, string $attribute): void
+    public function fillModelWithData($model, mixed $value, string $attribute): void
     {
         $lonLat = explode(',', $value);
         $value = $this->latLonToGeometry($lonLat);
