diff --git a/src/Fields/Field.php b/src/Fields/Field.php
index bd8ddf9..24ab695 100644
--- a/src/Fields/Field.php
+++ b/src/Fields/Field.php
@@ -480,7 +480,7 @@ abstract class Field extends FieldElement implements JsonSerializable, Resolvabl
      * @param  mixed  $value
      * @return void
      */
-    public function fillModelWithData($model, $value, string $attribute)
+    public function fillModelWithData(object $model, mixed $value, string $attribute): void
     {
         $attributes = [Str::replace('.', '->', $attribute) => $value];
 
